src/main/deploy/robotThing1.xml

```diff
@@ -1,499 +1,484 @@
-              currentLimiting="true" 
-              forwardlimitswitch="true"
-              forwardlimitswitchopen="true"
-              reverselimitswitch="true"
-              reverselimitswitchopen="true"/> 
-       <!-- digitalInput usage= "BALL_PRESENT"
-                     digitalId="1"
-                     reversed="false"/ -->
-  </mechanism>
-
-  <mechanism type="RIGHT_INDEXER"
-              networkTable="IndexerNT"
-              controlFile="indexer.xml">
-	<motor usage="INDEXER"
-              canId="7"
-              pdpID="7"
-              type="TALONSRX"
-              inverted="true"
-              sensorInverted="false"
-              feedbackDevice="NONE"
-              countsPerRev="0"
-              gearRatio="1.0"
-              brakeMode="true"
-              peakCurrentDuration="50"
-              continuousCurrentLimit="30"
-              peakCurrentLimit="40"
-              currentLimiting="true" 
-              forwardlimitswitch="true"
-              forwardlimitswitchopen="true"
-              reverselimitswitch="true"
-              reverselimitswitchopen="true"/>
-       <!-- digitalInput usage="BALL_PRESENT"
-                     digitalId="1"
-                     reversed="false"/ -->
-  </mechanism>
-
-<!-- ====================================================
-     Climber
-     ====================================================  -->
-   <mechanism type="CLIMBER"
-               networkTable="climberNT"
-               controlFile="climber.xml">
-       <motor usage="CLIMBER_LIFT"
-              canId="9"
-              pdpID="9"
-              type="FALCON"
-              inverted="false"
-              sensorInverted="false"
-              feedbackDevice="INTERNAL"
-              countsPerRev="2048"
-              gearRatio="1.0"
-              brakeMode="true"
-              peakCurrentDuration="50"
-              continuousCurrentLimit="30"
-              peakCurrentLimit="40"
-              currentLimiting="true" 
-              forwardlimitswitch="false"
-              forwardlimitswitchopen="true"
-              reverselimitswitch="false"
-              reverselimitswitchopen="true"/>
-
-       <motor usage="CLIMBER_ROTATE"
-              canId="20"
-              pdpID="15"
-              type="FALCON"
-              inverted="true"
-              sensorInverted="false"
-              feedbackDevice="INTERNAL"
-              countsPerRev="2048"
-              gearRatio="1.0"
-              brakeMode="true"
-              peakCurrentDuration="50.0"
-              continuousCurrentLimit="20.0"
-              peakCurrentLimit="25.0"
-              currentLimiting="true" 
-              forwardlimitswitch="false"
-              forwardlimitswitchopen="true"
-              reverselimitswitch="false"
-              reverselimitswitchopen="true"/>
-   </mechanism>
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE robot SYSTEM "robot.dtd">
+<robot>
+       <pdp canId="1"
```

Comment:
thing1 doesn't have a rev PDH, nor many of the other things in this file.   Why can't we just use robomechanismsonthings.xml? Or if we need specific files for the things, then it should list only what is actually there.